tests/cython/README.md

```diff
@@ -3,13 +3,9 @@
 
 ```nose``` Python testing framework is required.
 
-Installation using ```pip``` package manager :
+Installation using ```apt-get``` package manager :
 ```
-pip install nose
-```
-or install it using ```conda``` package manager:
-```
-conda install nose
+apt-get install python-nose
 ```
```

Comment:
Not to drive you crazy, but the above is for python 2.  There is also a python3-nose or a python-nose3 which can be installed at the same time, without conflicts ... however, I don't think our current cmakefiles are smart enough to use it.  Or something like that.  It might be why cmake is currently not launching the nose tests.

Reply:
Yes, you're right, the same thing with ```cython```. May be using ```apt-get``` is not a good idea because all of python packages in Ubuntu have two versions ```python``` and ```python3``` and you should always choose version which corresponds to your environments.

At the moment I use ```conda``` package management on my machine because it allows opaque switching between python2/python3 environments and have richer package base.